### ISSUE TITLE
visualization Grad_CAM

### ISSUE THREAD

**Issue by W4ngH4o** [Sense-X/UniFormer#79]
TENSORBOARD:
  ENABLE: True
  MODEL_VIS: 
    ENABLE: True
    MODEL_WEIGHTS: True
    ACTIVATIONS: True
    INPUT_VIDEO: True
    LAYER_LIST:  ?
    GRAD_CAM:
      ENABLE: True
      LAYER_LIST:  ?

Thanks so much. I want to know what to fill in  '?'  to apply Grad-CAM to show the areas of the greatest concern in the last layer.  The tensorboard result of the training process is normal,  but the grad_cam is blank.

**Comment by Andy1621** [comment]
Thanks for your question. Please check [PySlowfast](https://github.com/facebookresearch/SlowFast/blob/main/VISUALIZATION_TOOLS.md) for more help, since the code is provided by them.

**Comment by W4ngH4o** [comment]
@Andy1621 Thanks for your reply. I have referred the code PySlowfast, and I found something confusing.  Seemingly your code doesn't contain visualization.py and run_net.py only processes training and testing except visualizing.




![1111](https://user-images.githubusercontent.com/47959754/192479359-c0be8388-085d-4365-b6dc-18d84b9a0585.JPG)

![2222](https://user-images.githubusercontent.com/47959754/192479416-8129fb88-8cb9-4569-a868-28b18811b28c.JPG)
